### DISCUSSION TITLE
데이터 베이스로 시간표짤때

### DISCUSSION THREAD

**Discussion by MadeByZiNo** [Goranies/ProjectRepository#33] [General]
강의가 따로 나눠져있는 경우에는 둘 다 넣어도 되지만 (3시간 짜리를 월2시간 화 1시간 나눠 듣는 강의)

교수랑 강의자체는 같은데 시간만 다른 강의(아예 다른강의 A반 B반)은 같이 못넣게 해주세요.

**Comment by byungmeo** [comment]
데이터베이스엔 넣을 수 있어야됨


![image](https://user-images.githubusercontent.com/37506142/128307357-1983268d-de28-47a8-a9e6-56bbb3a08dbd.png)

**Reply by MadeByZiNo** [reply]
아 데이터베이스 말고 그 완성된 시간표

**Reply by byungmeo** [reply]
그냥 같은강의명 & 같은교수명이면 시간표에 안넣어지게 한다는 뜻이지?

**Reply by MadeByZiNo** [reply]
ㅇㅇ 그 완성된 시간표
